### PR TITLE
chore: move from polished to tinycolor2

### PR DESCRIPTION
**Pull request by sendilkumarn**
Replaces `polished` library with a light weight `tinycolor2`, to reduce the bundle size (a little). 

#### Scope

- [ ] Patch: Bug Fix
- [ ] Minor: New Feature
- [ ] Major: Breaking Change

### CODE REVIEW

**Comment by gergelyke** [comment]
thank you @sendilkumarn ! - can you please post a screenshot of the bundle size before/after?

**Comment by sendilkumarn** [comment]
> Before (using polished)

Screenshot 2019-10-29 at 18 25 13


> After (using tinyColor2)

Screenshot 2019-10-29 at 18 21 19


If possible, can we also document how to do this? It might help someone.

File: src/tag/styled-components.js [hunk 1]
```diff
@@ -48,6 +49,14 @@ function fontColor(props, isHovered?: boolean, isActionText?: boolean) {
...
         return props.$theme.colors.tagNegativeFontDisabled;
       case KIND.custom:
+        console.log(
+          props.$color,
+          props.$theme.colors.tagFontDisabledRampUnit,
+          customOnRamp(
+            props.$color,
+            props.$theme.colors.tagFontDisabledRampUnit,
+          ),
+        );
```
Comment (sandgraham) [inline src/tag/styled-components.js]: need to remove this `console.log`

File: src/tag/styled-components.js [hunk 2]
```diff
@@ -48,6 +49,14 @@ function fontColor(props, isHovered?: boolean, isActionText?: boolean) {
       case KIND.negative:
         return props.$theme.colors.tagNegativeFontDisabled;
       case KIND.custom:
+        console.log(
```
[the remark this answers is not in the document]
Comment (sendilkumarn) [reply src/tag/styled-components.js]: oops sorry didn't remove this after testing

**Comment by sandgraham** [comment]
@sendilkumarn There is a fix for some of the `vrt` (visual regression test) issues on `uber/baseweb:master` now. If you rebase or merge in latest, CI should run correctly. Thanks and sorry for the inconvenience.

**Comment by sendilkumarn** [comment]
@sandgraham no worries updated the PR

**Comment by kahwee** [comment]
Another alternative to switching to `tinycolor2` is to use:

```
import darken from 'polished/lib/color/darken';
```

What do you think?

**Comment by gergelyke** [comment]
just put up a diff for that too - https://github.com/uber/baseweb/pull/2216

@sendilkumarn can you try this in your project too?

**Comment by gergelyke** [comment]
for now, landed https://github.com/uber/baseweb/pull/2216 instead - please let us know if that helped

**Pull request closed without merging**